### ISSUE TITLE
Script runs, but spits out errors.  No data created.

### ISSUE THREAD

**Issue by TnTBass** [VeeamHub/grafana#3]
**Describe the bug**
Running the veeam-availability-console-script.sh script as exists errors out with 400 Bad Request errors.

**To Reproduce**
1. Run Script Manually
2. Check output.

**Expected behavior**
In the blog it states I should see success messages.  

**Errors**
HTTP/1.1 400 Bad Request
Content-Type: application/json
Request-Id: 20c738b7-4f66-11ea-83de-005056b73015
X-Influxdb-Build: OSS
X-Influxdb-Error: unable to parse 'veeam_vac_vcclicense,edition=,status=,type=,contactPerson=,backupServerName=,companyName= licenseExpirationDate=1581663600,supportExpirationDate=1581663600,licensedCloudConnectBackups=,usedCloudConnectBackups=,licensedCloudConnectReplicas=,usedCloudConnectReplicas=,licensedCloudConnectServers=,usedCloudConnectServers=,licensedCloudConnectWorkstations=,usedCloudConnectWorkstations=': missing tag value
X-Influxdb-Version: 1.7.10
X-Request-Id: 20c738b7-4f66-11ea-83de-005056b73015
Date: Fri, 14 Feb 2020 20:11:04 GMT
Content-Length: 433

**Versions**
InfluxDB: 1.7.10
VAC: 
Server Version 3.0.0.2795
Web UI Version 3.0.0.2795
Grafana: 6.6.1

**Comment by jorgedlcruz** [comment]
Let me look into this, probably it is related that the script it has been not updated to VAC v3, just yet.

**Comment by TnTBass** [comment]
It looks like its not assigning the Bearer token properly. Walking through each line manually I get nothing for the $Bearer variable.
`Bearer=$(curl -X POST --header "Content-Type: application/x-www-form-urlencoded" --header "Accept: application/json" -d "grant_type=password&username=$Username&password=$Password" "$RestServer:$RestPort/token" -k --silent | jq -r '.access_token')
`

**Comment by TnTBass** [comment]
I managed to get past that error, I needed to enter https:// in front of my Rest Server entry.  I am getting data populated, but its not all correct. 

For example:
![Annotation 2020-02-18 155240](https://user-images.githubusercontent.com/1081294/74785193-f774f300-5266-11ea-8929-4bcac925191f.png)
License count is obviously wrong, also the contact person should have something (but maybe this is missing from my VAC?  Not sure where to validate the source).

Also, on the tenant side, I see this kind of thing:
![Annotation 2020-02-18 155559](https://user-images.githubusercontent.com/1081294/74785457-9c8fcb80-5267-11ea-9291-53d58758c91f.png)
Expiration date, again obviously wrong.  The "Only queries that return single series/table is..." is not terribly helpful either.
